File: 프로그래머스/unrated/120812. 최빈값 구하기/최빈값 구하기.js
function solution(array) {
    const count = {};
    array.forEach(el => count[el] ? count[el]++ : count[el] = 1)
    
    const keys = Object.keys(count)
    const values = Object.values(count)
    const max = Math.max(...values)
    const maxs = keys.filter(el => count[el] === max)
    return maxs.length >= 2 ? -1 : Number(maxs)
}

File: 프로그래머스/unrated/120812. 최빈값 구하기/최빈값 구하기.py
def solution(array):
    keys = [];
    count = [];

    for i in range(len(array)):
        if array[i] not in keys:
            keys.append(array[i]);
            
    for i in range(len(keys)):
        count.append(array.count(keys[i]));
        
    return -1 if count.count(max(count)) >= 2 else keys[count.index(max(count))]